tests/fsharp/typecheck/sigs/neg16.bsl

```diff
@@ -63,12 +63,16 @@ but here has type
 
 neg16.fs(85,8,85,18): typecheck error FS0039: The pattern discriminator 'FooA++' is not defined.
 
+neg16.fs(85,7,85,22): typecheck error FS0025: Incomplete pattern matches on this expression.
```

Comment:
I know this is minor, but the code for this test:
```fsharp
  let (|``FooA++``|) (x:int) = x
  let (``FooA++``(x)) = 10
```

Normally it would error saying that `FooA++` is not defined because using "++" in the name is not allowed. But now it will include an additional error saying that the pattern matches are incomplete, but it isn't a valid match construct to begin with.

Again, this is minor and not a blocker.